test/api/games.js

```diff
@@ -1,26 +1,31 @@
-const expect = require('chai').expect;
-
-const request = require('superagent');
-const dataMakers = require('../helpers/data');
+import { expect } from 'chai';
+import { createUserGroupGameRelease, API_GAMES_URL } from '../helpers';
+import fetch from 'node-fetch';
 
 describe('api/games', () => {
   describe('GET', () => {
-    it('should receive a list of all games with a production release, without requiring a token', async function() {
-      await dataMakers.makeGame('prod');
-      let response = await request.get('http://localhost:3000/api/games');
-      expect(response.body.success).to.equal(true);
-      expect(response.body.data.length).to.equal(1);
-      expect(response.body.data[0].releases.length).to.equal(1);
+    it('should receive a list of all games with a production release, without requiring a token', async () => {
+      await createUserGroupGameRelease();
+      const response = await fetch(API_GAMES_URL).then(
+        async r => await r.json()
+      );
```

Comment:
Really I just swapped it because I'm more familiar with fetch, and I wanted to remove all unknowns while trying to a debug an issue that was causing it to fail some times. Although to be honest I'd prefer to use fetch over other http libraries as it's a standardized way to make http requests in Javascript